app/src/Components/Account/PasswordInput.jsx

```diff
@@ -0,0 +1,33 @@
+import EyeSlashIcon from "./icons/EyeSlashIcon.jsx";
+import EyeIcon from "./icons/EyeIcon.jsx";
+import React, {useState} from "react";
+
+function PasswordInput (props) {
```

Comment:
Please specify the type of the props.

Reply:
chez moi c'est Used, avant de le register, maintenant dans le Form